docker-compose-core.yaml

```diff
@@ -91,9 +91,28 @@ services:
     networks:
       - camunda-platform
 
+  kibana:
+    image: docker.elastic.co/kibana/kibana:${ELASTIC_VERSION:-7.17.0}
+    container_name: kibana
+    ports:
+      - 5601:5601
+    environment:
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
```

Comment:
Why would you do this for kibana? I know the reason for elasticsearch, but Kibana is just a normal webapp?